docs/releasing.md

```diff
@@ -1,17 +1,25 @@
 # Release process
 
+## Semi-automatic
+
+1. make the release artifacts `make release artifacts`
```

Comment:
I'm not sure about forcing a `clean`, but should use `release-artifacts` instead of `release artifacts`

Reply:
```suggestion
1. make the release artifacts `make release-artifacts`
```